File: java/trip-service-kata/main/java/org/craftedsw/harddependencies/trip/TripDAO.java
package org.craftedsw.harddependencies.trip;

import java.util.List;

import org.craftedsw.harddependencies.exception.DependendClassCallDuringUnitTestException;
import org.craftedsw.harddependencies.user.User;

public class TripDAO {

	public static List<Trip> findTripsByUser(User user) {
		throw new DependendClassCallDuringUnitTestException(
				"TripDAO should not be invoked on an unit test.");
	}
	
}

File: python-objects/TripDAO.py
from DependendClassCallDuringUnitTestException import DependendClassCallDuringUnitTestException


class TripDAO(object):
    @staticmethod
    def find_trips_by_user(user):
        raise DependendClassCallDuringUnitTestException("TripDAO should not be invoked on an unit test.")
